runtime/src/bank.rs

```diff
@@ -193,6 +193,8 @@ mod address_lookup_table;
 mod builtin_programs;
 mod metrics;
 mod sysvar_cache;
+#[cfg(test)]
+pub(crate) mod tests;
```

Comment:
```suggestion
pub(self) mod tests;
```

`self` _should_ be sufficient

Reply:
there's 1 case where `bank_forks` reaches in here to call `update_vote_account_timestamp`

Reply:
oh god...  maybe pop it out of the `tests` module, then?